config/application.rb

```diff
@@ -153,7 +153,14 @@ class Application < Rails::Application
 
     # enables in memory cache store with ttl
     # config.cache_store = TimedCachedStore.new
-    config.cache_store = :file_store, Rails.root.join("tmp", "cache")
+    if SETTINGS[:cache_store] == 'redis'
+      args = [:redis_cache_store]
+      Array.wrap(SETTINGS[:redis_hosts]).each { |h| args << h }
```

Comment:
Is there a default host that is supported out of the box without having to configure it? If not could we add a default localhost if none other is provided?

Reply:
Rails defaults to `redis://localhost:6379/0`.

Reply:
Works out of box, tested.